docs/source/models.rst

```diff
@@ -677,54 +671,6 @@ accepts the following data formats as input:
     response_json = json.loads(response.text)
     print(response_json)
 
-.. rubric:: Example workflow using the MLflow CLI
```

Comment:
I don't want to remove these things until the replacement command is in place - wdyt?

Reply:
yes definitely agree, I'll remove the reference

Reply:
@akshaya-a I agree that we should keep these CLI docs here until we have a replacement CLI. @shivp950 can you restore this CLI example and indicate that it uses the deprecated `mlflow azureml build-image` CLI & that a replacement is coming soon?